src/server/game/Miscellaneous/Language.h

```diff
@@ -666,8 +666,14 @@ enum AcoreStrings
     LANG_YOUR_BG_LEVEL_REQ_ERROR        = 715,
     //                                      = 716, see LANG_PINFO_MAP_OFFLINE
     LANG_BG_STARTED_ANNOUNCE_WORLD      = 717,
-    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_JOIN = 718,
-    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_EXIT = 719,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_JOIN_NAME_RATING = 718,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_EXIT_NAME_RATING = 719,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_JOIN_NAME = 753,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_EXIT_NAME = 754,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_JOIN_RATING = 755,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_EXIT_RATING = 756,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_JOIN = 757,
+    LANG_ARENA_QUEUE_ANNOUNCE_WORLD_EXIT = 758,
```

Comment:
Ideally, these should be placed in numeric order for the whole file to not mistakenly do double entries or remove stuff.